component/web/oauth-common/src/main/java/org/gatein/security/oauth/openid/OpenIdProcessorImpl.java

```diff
@@ -106,11 +114,12 @@ public OpenIdProcessorImpl(ExoContainerContext context, InitParams params, Secur
         }
 
         this.secureRandomService = secureRandomService;
+        this.remoteJwkSigningKeyResolver = new RemoteJwkSigningKeyResolver(this.oidcServer);
```

Comment:
Not needed here. The signingKeyResolver will be use only for validate the jwt token. 
I dont think we need it in this task, perhaps in the tird one, but not here